docs-site/content/guide/laravel-full-text-search.md

```diff
@@ -0,0 +1,567 @@
+     */
+    protected $casts = [
+        "release_date" => "datetime",
+        "price" => "float",
+        "positive" => "integer",
+        "negative" => "integer",
+        "app_id" => "integer",
+        "min_owners" => "integer",
+        "max_owners" => "integer",
+        "hltb_single" => "integer",
+    ];
+
+}
+```
+
+</template>
+
+</Tabs>
+
+And then define the <RouterLink :to="`/${$site.themeConfig.typesenseLatestVersion}/api/collections.html#with-pre-defined-schema`">Collection Schema</RouterLink> in the `config/scout.php` file:
+
+<Tabs :tabs="['PHP']">
+
+<template v-slot:PHP>
+
+```php{3,9-70}
+<?php
+
+use App\Models\Game;
+
+return [
+  ...
+    'typesense' => [
+    ...
+        'model-settings' => [
+            Game::class => [
+                "collection-schema" => [
+                    "fields" => [
+                        [
+                            "name" => "name",
+                            "type" => "string",
+                        ],
+                        [
+                            "name" => "price",
+                            "facet" => true,
+                            "type" => "float",
+                        ],
+                        [
+                            "name" => "hltb_single",
+                            "type" => "int32",
+                            "facet" => true,
+                            "optional" => true,
+                        ],
+                        [
+                            "name" => "positive",
+                            "facet" => true,
+                            "type" => "int32",
+                        ],
+                        [
+                            "name" => "negative",
+                            "facet" => true,
+                            "type" => "int32",
+                        ],
+                        [
+                            "name" => "app_id",
+                            "type" => "string",
+                        ],
+                        [
+                            "name" => "min_owners",
+                            "type" => "int32",
+                        ],
+                        [
+                            "name" => "max_owners",
+                            "type" => "int32",
+                        ],
+                        [
+                            "name" => "created_at",
+                            "type" => "int64",
+                        ],
+                        [
+                            "name" => "release_date",
+                            "type" => "int64",
+                        ],
+                    ],
+                    "default_sorting_field" => "release_date",
+                ],
+                "search-parameters" => [
+                    "query_by" => "name, app_id",
+                ],
+            ],
+        ],
+    ],
+```
+
+</template>
+
+</Tabs>
+
+After which, you can import the data into Typesense by running the following command:
```

Comment:
Could you add a section title here saying this is for Backfilling existing data. 

And just before that section, we want to say that once the Laravel Scout Typesense driver is setup, all changes to the models will be synced to Typesense automatically. 

And may be provide a code sample that shows modifying a DB record, and showing that it is reflected in Typesense by doing a search.

Reply:
The first issue is resolved in 72ecaa6185509485296c72359d080d49d8577b8c

About mentioning auto-syncing, it's being added in 8f267752fb9292da874edfac8da3127a72c17cbe

And finally, I added a small part showcasing real-time syncing through inserting a new record in the database in 2ae61d1c462d2a772770c67363546228b8c77c2b